R/derive_vars_disposition_reason.R

```diff
@@ -170,9 +170,8 @@ derive_disposition_reason <- function(dataset,
 #' if required (e.g. `DSTERM`).
 #'
 #' @details
-#' `format_reason_default(DSDECOD)` returns `DSDECOD` when `DSDECOD` is not
-#' 'COMPLETED' nor `NA`. `format_reason_default(DSDECOD, DSTERM)` returns
-#' `DSTERM` when `DSDECOD` is not 'COMPLETED' nor `NA`.
+#' `format_reason_default(DSDECOD)` returns `DSDECOD` when `DSDECOD` is not `'COMPLETED'` nor `NA`.
+#' \cr`format_reason_default(DSDECOD, DSTERM)` returns `DSTERM` when `DSDECOD` is not `'COMPLETED'` nor `NA` and `DSDECOD` is equal to `'OTHER'`.
```

Comment:
Could we simplify it by removing "`DSDECOD` is not `'COMPLETED'` nor `NA` and "?

Reply:
Updated.